src/js/index.js

```diff
@@ -63,13 +63,14 @@ function ID3Tag() {
     }
 }
 
-function Mp3Encoder(channels, samplerate, kbps) {
-    if (arguments.length != 3) {
-        console.error('WARN: Mp3Encoder(channels, samplerate, kbps) not specified');
-        channels = 1;
-        samplerate = 44100;
-        kbps = 128;
-    }
+function Mp3Encoder(config) {
+    config = Object.assign({
```

Comment:
overall the idea is good, but i think it makes sense to keep backwards compatible constructor for current users of the api
`if (typeof arguments[0] === 'object') { ... your code ... } else { ... backwards compatible constructor ... }`